10_PredictionExamples.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>This is super short. You should be way more descriptive here -- say that we'll be using a simple linear regression model. Define the true population model, which is y = 2 * x + epsilon.</p><p> </p><p>I'd also pick a much shorter header, something like "A linear model".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>